client-v2/src/shared/util/shared.ts

```diff
@@ -123,6 +124,18 @@ const addGroups = (
     { live: [], draft: [], previously: [], addedGroups: {} } as ReduceResult
   );
 
+const createPreviouslyArticleFragmentIds = (
```

Comment:
Is it worth adding some explanation of why we're doing this? Without context, it probably looks a bit unusual. Maybe something like:

```suggestion
// To determine the UUIDs of article fragments recently removed from a collection in a way that
// preserves the overall ordering lost during normalisation (when fragments are assigned to a Group)
// we need to compare both the pre-normalised and normalised versions of the same collection.
const createPreviouslyArticleFragmentIds = (
```

Reply:
sure thing